CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased changes
```

Comment:
Should the v0.1 section be filed under "Unreleased changes"? It's on PyPI right?

Reply:
It's a sibling, rather than a subheading. I think this is OK because this is a living document and new changes are likely to be added to this section a lot.

This file doesn't get sent to PyPI directly -- instead there's a link to the file in this repo.